external_review/export_DO.sh

```diff
@@ -5,8 +5,13 @@ function export_DO {
     echo $2
     geography_level=$1
     category=$2
-    filename="${category}_${geography_level}.csv"
-    SPACES="spaces/edm-publishing/db-eddt/${category}"
+    local filename="${category}_${geography_level}.csv"
+    local branchname=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD)
```

Comment:
This looks good to me! I personally want to do more to improve the current export workflow but maybe this isn't the right time for that